src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
```

Comment:
This should also be defined at the top of the file: https://www.python.org/dev/peps/pep-0008/#constants

Reply:
That's one argument.  The other is having it close the code that it's relevant to and not having to jump around the file.  If it were tripping a pylint trigger, I'd be more inclined to move it.